app/code/Transoft/Blog/Controller/Adminhtml/Index/Upload.php

```diff
@@ -0,0 +1,56 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Controller\Adminhtml\Index;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Catalog\Model\ImageUploader;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Framework\Controller\ResultInterface;
+
+/**
+ * Class Upload
+ *
+ * Upload controller class
+ */
+class Upload extends Action implements HttpGetActionInterface
```

Comment:
Why is **upload** a **GET** action? Also as I can see this is almost copy of \Magento\Catalog\Controller\Adminhtml\Category\Image\Upload. Why can't we just extend from it?

Reply:
fixed